src/services/db/GitFileSystemService.ts

```diff
@@ -984,12 +1011,18 @@ export default class GitFileSystemService {
   ): ResultAsync<DirectoryContents, GitFileSystemError | NotFoundError> {
     const isStaging = this.isStagingFromBranchName(branchName)
 
-    return this.listDirectoryContents(repoName, directoryPath, branchName)
+    return this.listDirectoryContents(
+      repoName,
+      directoryPath,
+      branchName,
+      false
+    )
       .andThen((directoryContents) =>
         okAsync(
           getPaginatedDirectoryContents(directoryContents, page, limit, search)
         )
       )
+      .andThen((dirContents) => okAsync(dirContents))
```

Comment:
hey i think this was accidentally added!

Reply:
will remove